web-ng/src/app/components/inline-edit-title/inline-edit-title.component.scss

```diff
@@ -31,19 +33,19 @@ input {
   border: none;
   font-size: 14pt;
   margin: 10px 10px 5px 0;
-  border: 1px solid transparent;
-  color: #afa7a7
+  border: 2px solid transparent;
```

Comment:
why is that we are using mat-color for some styling and not for others?

Reply:
In cases where there is no color code like here it's because we're allowing the default global style to work for us. In cases where the hex code is still used it's just because I haven't gotten to converting them yet :)

Reply:
sounds good :)